src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts

```diff
@@ -52,20 +58,26 @@ export class HaDeviceInfoZWaveJS extends LitElement {
     if (!this._nodeId || !this._entryId) {
       return;
     }
+
+    const configEntries = await getConfigEntries(this.hass);
+    this._configEntry = configEntries.find(
+      (entry) => entry.entry_id === this._entryId!
+    );
+
     this._node = await fetchNodeStatus(this.hass, this._entryId, this._nodeId);
   }
 
   protected render(): TemplateResult {
-    if (!this._node) {
+    if (!this._node || !this._configEntry) {
       return html``;
     }
     return html`
       <h4>
         ${this.hass.localize("ui.panel.config.zwave_js.device_info.zwave_info")}
       </h4>
       <div>
-        ${this.hass.localize("ui.panel.config.zwave_js.common.home_id")}:
-        ${this._homeId}
+        ${this.hass.localize("ui.panel.config.zwave_js.common.source")}:
+        ${this._configEntry.title}
```

Comment:
Should we even show this if there is only 1 zwave js config entry?

Reply:
Ooo good idea. Definitely not necessary if there's only 1 instance of zwavejs set up